spec/inputs/eventlog_spec.rb

```diff
@@ -1,7 +1,11 @@
 require "logstash/devutils/rspec/spec_helper"
-require 'logstash/inputs/eventlog'
```

Comment:
can we might be add here some logging/feedback to stdout in order to provide some feedback in case of the platform being non windows. I understand using tags will simply filter out the test, but the user deserve some feedback to be aware why the test are not run. what do you think?